go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -70,21 +74,64 @@ type serviceConfig struct {
 	service *kapi.Service
 	// Were those rules etp:local + Host Networked rules
 	etpHostRules bool
+	// Used when svc=etp:local + it has non local host networked backends in LGW.
+	// NOTE: For most part this is always the opposite of etpHostRules, however there
+	// can be a situation where there are no endpoints for a service in which case
```

Comment:
what is the situation?

Reply:
example, an idled service or a case where only svc is created and backends are not yet ready so 0 endpoints. Or a case where someone accidentally deleted the endpoints? Basically situations where endpoints.Subset is an empty list? 

Is it really ok to label that as "opposite of local-host-networked-endpoints" in these situations and then let the load-balancer controller take action - which it already does btw, the backend strings on the load balancer become "" (empty) so packet gets rejected?